lib/Targets/AIETargets.cpp

```diff
@@ -389,11 +416,15 @@ SECTIONS
               doBuffer(tile, target_model.getMemEastBaseAddress(),
                        std::string("east"));
 
+            int stacksize = 0;
+            if (auto core = tile.getCoreOp())
+              stacksize = core.getStackSize();
             output << "_stack    DM_stack 0x"
                    << llvm::utohexstr(
                           target_model.getMemInternalBaseAddress(srcCoord))
-                   << "  0x400 //stack for core\n"; // TODO Needs to match
-                                                    // stack size!!!
+                   << "  0x" << llvm::utohexstr(stacksize)
```

Comment:
Did you find a way to put the stack at a symbolic location to maximize the stack + heap usage? It looks like Bridge linker is very limited and does not have the concept of "." for example.